buildSrc/build.gradle

```diff
@@ -10,17 +10,17 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.diffplug.spotless:spotless-plugin-gradle:6.11.0'
+    implementation 'com.diffplug.spotless:spotless-plugin-gradle:6.20.0'
     implementation 'com.felipefzdz.gradle.shellcheck:shellcheck:1.4.6'
-    implementation 'com.google.cloud.tools.jib:com.google.cloud.tools.jib.gradle.plugin:3.3.0'
+    implementation 'com.google.cloud.tools.jib:com.google.cloud.tools.jib.gradle.plugin:3.3.2'
     implementation 'com.srcclr.gradle:com.srcclr.gradle.gradle.plugin:3.1.12'
-    implementation 'de.undercouch.download:de.undercouch.download.gradle.plugin:5.2.0'
-    implementation 'com.github.spotbugs.snom:spotbugs-gradle-plugin:5.0.12'
-    implementation 'io.spring.dependency-management:io.spring.dependency-management.gradle.plugin:1.0.14.RELEASE'
+    implementation 'de.undercouch.download:de.undercouch.download.gradle.plugin:5.4.0'
+    implementation 'com.github.spotbugs.snom:spotbugs-gradle-plugin:5.0.14'
+    implementation 'io.spring.dependency-management:io.spring.dependency-management.gradle.plugin:1.1.2'
     implementation 'org.hidetake.swagger.generator:org.hidetake.swagger.generator.gradle.plugin:2.19.2'
-    implementation 'org.sonarqube:org.sonarqube.gradle.plugin:3.4.0.2513'
+    implementation 'org.sonarqube:org.sonarqube.gradle.plugin:4.3.0.3225'
```

Comment:
With this upgrade, I think the new sonar command changes from "sonarqube" to "sonar". 
https://github.com/DataBiosphere/java-pfb/blob/main/buildSrc/src/main/groovy/bio.terra.pfb.java-common-conventions.gradle#L68

Reply:
I don't think I ever found anything in the docs to confirm that it changed with v4, but the suggested sonar code provided when setting up java-pfb used "sonar" instead of "sonarqube". All of the rest of the configurations remained the same.

Reply:
Good point, I'll update that. If you look carefully at the noisy output, there is a deprecation warning:
```
Task 'sonarqube' is deprecated. Use 'sonar' instead.
```

Reply:
Oh great, I'm glad you found confirmation!

Reply:
I've updated the PR and replaced `sonarqube` with `sonar` in a few places; it seem like `sonar` is now the general product name and `sonarqube` is specifically the enterprise scanning service.